XenAdmin/TabPages/GeneralTabLicenseStatusStringifier.cs

```diff
@@ -74,20 +74,20 @@ public string ExpiryStatus
                         return Messages.LICENSE_UNLICENSED;
 
                     if (Status.CurrentState == LicenseStatus.HostState.Free)
-                        return Messages.LICENSE_UNLICENSED;
+                        return Helpers.NileOrGreater(Status.LicensedHost) ? Messages.GENERAL_UNKNOWN : Messages.LICENSE_UNLICENSED;

```

Comment:
Is this unkown or never?

Reply:
oh what about naming the field "License Status" instead of "Expiry Status"?


That way I can keep "Licensed" and "Not Found", and use "Trial License" for post Nile?

Reply:
I presume the reason why the original key was "Status" is because it's already under the licence details title, so it's known we refer to Licence status. Also note that it's Status on the licence manager too, so it's probably good to use uniform terminology.

"Not found" might imply "error trying to deduce the status" - Trial License sounds better, is the change too big/risky?

Reply:
> I presume the reason why the original key was "Status" is because it's already under the licence details title, so it's known we refer to Licence status. 

good point. will revert to Status

> "Not found" might imply "error trying to deduce the status" - Trial License sounds better, is the change too big/risky?

No I'll just add a new string for post Nile, rest will stay the same 👍